scripts/derivechangeset

```diff
@@ -0,0 +1,7 @@
+
+# HootApiDb requires a map layer, though (hence, 'blah').
+step1:
+	hoot derive-changeset $(HOOT_OPTS) -D convert.bounding.box=$(aoi) -D osm.changeset.sql.writer.generate.new.ids=false $(input1) $(input2) $(output) $(jobid) $(DB_URL)/blah
+
```

Comment:
No, the url is optional from the command line.  You can still do two file inputs that way....just the services were set up initially to only to work with db layers since there isn't a file input use case for the services.